### PR TITLE
Order changes

### PR DESCRIPTION
**Pull request by vladislav-kozachenko**
In this small PR we have changed all classes that relate to the order aggregate as follows:
- Corrected mistakes in Javadoc
- Used static imports if it possible
- Updated display names for tests (concluded classes and methods in backticks)
- Used package private access modifiers for order utility classes methods
- Moved utility methods from `OrderAggregate` to `Orders`

### CODE REVIEW

File: api-java/src/main/java/javaclasses/mealorder/c/order/OrderAggregate.java [hunk 1]
```diff
@@ -187,11 +175,9 @@ OrderCanceled handle(CancelOrder cmd) throws CannotCancelProcessedOrder {
...
      *
-     * <p>
-     * Creates new order and changes order aggregate state status on {@code ORDER_ACTIVE}.
-     * Checks if the order with the same id was already created and cancelled. In this case
+     * <p> Creates new order and changes order aggregate state status on {@code ORDER_ACTIVE}.
+     * Checks if the order with the same ID was already created and cancelled. In this case
      * removes all dishes that was added before cancellation.
-     * </p>
      *
      * @param event
```
Comment (YuriiHaidamaka) [inline api-java/src/main/java/javaclasses/mealorder/c/order/OrderAggregate.java]: add the description of return type

File: api-java/src/main/java/javaclasses/mealorder/c/order/OrderAggregateRejections.java [hunk 2]
```diff
@@ -72,38 +72,38 @@ private CreateOrderRejections() {
...
+        static OrderAlreadyExists orderAlreadyExists(CreateOrder cmd) throws OrderAlreadyExists {
             checkNotNull(cmd);
             final OrderId orderId = cmd.getOrderId();
             final Timestamp timestamp = getCurrentTime();
-            return new OrderAlreadyExists(orderId, timestamp);
+            throw new OrderAlreadyExists(orderId, timestamp);
         }
 
         /**
          * Constructs and returns the {@link MenuNotAvailable} rejection
```
Comment (YuriiHaidamaka) [inline api-java/src/main/java/javaclasses/mealorder/c/order/OrderAggregateRejections.java]: Constructs and throws. Add this in the documentation for each method

File: api-java/src/main/java/javaclasses/mealorder/c/order/Orders.java [hunk 3]
```diff
@@ -70,11 +75,11 @@ public static boolean checkRangeIncludesDate(MenuDateRange range, LocalDate orde
     /**
      * Finds vendor for order.
      *
-     * @param orderId id of the order for which vendor have to be found.
+     * @param orderId ID of the order for which vendor have to be found.
      * @return Optional of vendor.
```
Comment (YuriiHaidamaka) [inline api-java/src/main/java/javaclasses/mealorder/c/order/Orders.java]: Optional of Vendor.

File: api-java/src/main/java/javaclasses/mealorder/c/order/Orders.java [hunk 4]
```diff
@@ -114,4 +118,17 @@ public static void checkMenuAvailability(CreateOrder cmd) throws
             throw menuNotAvailable(cmd);
```
Comment (YuriiHaidamaka) [inline api-java/src/main/java/javaclasses/mealorder/c/order/Orders.java]: 115 line : Extract menu.get().getMenuDateRange() and if statement will be in one line

File: api-java/src/test/java/javaclasses/mealorder/c/order/AddDishToOrderTest.java [hunk 5]
```diff
@@ -143,7 +144,7 @@ void notAddDish() {
     }
```
Comment (YuriiHaidamaka) [inline api-java/src/test/java/javaclasses/mealorder/c/order/AddDishToOrderTest.java]: Make lines 140 -141   142-143  in one separate  line.

File: api-java/src/main/java/javaclasses/mealorder/c/order/Orders.java (hunk 4 above)
Comment (vladislav-kozachenko) [reply api-java/src/main/java/javaclasses/mealorder/c/order/Orders.java]: Impossible without addition another if Statement with the same body.
`menu.get().getMenuDateRange()` called after `menu.isPresent()` in one condition.

**Pull request merged**